hail/python/hail/linalg/blockmatrix.py

```diff
@@ -233,8 +235,21 @@ class BlockMatrix(object):
 
     - Natural logarithm, :meth:`log`.
     """
-    def __init__(self, jbm):
-        self._jbm = jbm
+    def __init__(self, bm):
```

Comment:
I prefer the argument type not be overloaded like this.  It should only take a BlockMatrix IR and you should have a BlockMatrix._from_java.

Reply:
Done